upgrade-executor/tasks/transfer-ownership.ts

```diff
@@ -0,0 +1,58 @@
+import "@nomiclabs/hardhat-ethers";
+import { ethers } from "ethers";
+import { IContractsTransfer } from "../utils/config";
+import { Ownable__factory } from "../typechain-types";
+import { AccessControlUpgradeable__factory } from "../typechain-types";
+
+export async function transferOwnership(
+  settings: IContractsTransfer,
+  newOwner: string,
+  signer: ethers.Signer
+) {
+  try {
+    // Transfer ownership of each contract in contracts
+    for (const [key, address] of Object.entries(settings.contracts)) {
```

Comment:
It seems logic for transfering ownership of "contracts" and "nonupgradeable" is the same. Do we need both?

Reply:
Yes, We can have sections like ownable and accesscontrol

Reply:
fixed